docs/podman-system-migrate.1.md

```diff
@@ -11,6 +11,11 @@ podman\-system\-migrate - Migrate container to the latest version of podman
 
 **podman system migrate** takes care of migrating existing containers to the latest version of podman if any change is necessary.
 
+Rootless Podman uses a pause process to keep the unprivileged namespaces alive.  This prevents any change to the `/etc/subuid`
+and `/etc/subgid` files to be propagated to the rootless containers when the pause process is already running.
+For these changes to be honored, it is necessary to stop all running containers and also stop the pause process.  Instead of doing it
+manually, **system migrate** can be used to stop the running containers and recreate the user namespace with the new configured mappings.
```

Comment:
I think this would be better if it said
```
manually. **podman system migrate** can be used to stop the running containers and recreate the user namespace with the new configured mappings.
``

Reply:
This should also be documented in troubleshooting section.  Although I don't know how often this situation arises.

I take if from your description that `podman system migrate` will also do a `podman container stop --all --force`

Reply:
thanks, fixed now.  It is already present in the troubleshooting file